randutil/crypto_test.go

```diff
@@ -45,3 +48,81 @@ func (s *cryptoRandutilSuite) TestCryptoToken(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(b, HasLen, 5)
 }
+
+var (
+	KernelTestUUID = "1031319a-b661-4c01-aafa-6def8a118944"
+)
+
+func MockKernelUUIDPath(newPath string) (restore func()) {
```

Comment:
(nitpick) In snapd the `MockFoo` is usually in `export_test.go` directly. It's not a requirement and I think we don't always follow the pattern but it seems appropriate here (but fine to keep if there is a reason to do it this way that I miss).